source/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "node-fetch";
+declare module "parse-diff";
+declare module "lodash.includes";
+declare module "lodash.find";
+declare module "jest-runtime";
+declare module "jest-environment-node";
+
+declare module "*/package.json";
```

Comment:
see what I did in line 8, which covered by TS behavior above link. Point is you can specify ambient declaration with wildcard, *and add necessary hint if you want*. so

`declare module "*/package.json"` -> `import * as pkg 'package.json' //works`

such as https://github.com/danger/danger-js/pull/78/files#diff-4b47fd8c4ebfbe9a6c8992df39f6f64dR4 .